### DISCUSSION TITLE
Where did ReactiveUI .net Maui ReactiveFlyoutPage go?

### DISCUSSION THREAD

**Discussion by DanielCauser** [reactiveui/ReactiveUI#3719] [Q&A] [answered]
Hello, I've been using reactiveui and ReactiveFlyoutPage on XamarinForms has been something I relied on for a while. Now that I migrated my codebase to Maui, I cannot find ReactiveFlyoutPage anymore, does anyone know what happened to it?

**Comment by ChrisPulman** [comment]
I will check if the maui base has a FlyoutPage and add this if it's there.

**Comment by ChrisPulman** [comment, marked as answer]
This will be fixed with #3720
